dubbo-dependencies-bom/pom.xml

```diff
@@ -478,6 +480,18 @@
                 <version>${activation_version}</version>
             </dependency>
 
+            <!-- for dubbo-configcenter-etcd -->
```

Comment:
It is better to put all the modules to the `Test lib`. This make is clear that these are just for testing purpose.

Reply:
sorry, so where it should be? I found embedded-redis for test also placed in this dependencies-bom xml, and found no where else to put this dependency.

Reply:
oh, I got your idea!